src/chi/components/sidebar/sidenav.scss

```diff
@@ -55,39 +54,161 @@ $sidenav-lg-width: 12.5rem;
         line-height: 1rem;
         text-align: center;
       }
-
-      &:hover {
-        color: $primary-color;
-      }
     }
 
     &.-active {
       > a {
+        background-color: rgba(set-color(blue, 70), 0.08);
         box-shadow: inset 0.125rem 0 0 $primary-color;
         color: $primary-color;
         font-weight: 600;
+
+        &.-unselected {
+          background-color: transparent;
+          box-shadow: none;
+        }
       }
     }
 
-    &.-active > a { // List item -active. The one that has the left 3px blue border.
-      background-color: set-color(grey, 20);
+    &:not(.-active) {
+      a.-active {
+        background-color: set-color(grey, 10);
+        box-shadow: inset 0.125rem 0 0 set-color(grey, 100);
+      }
     }
 
     & > a {
       &.-active { // Drawer shown, will sets active the trigger (this anchor element)
-        background-color: set-color(grey, 20);
+        background-color: set-color(grey, 10);
       }
     }
+  }
 
-    &:not(.-active) > a:not(.-active):hover {
-      background-color: set-color(grey, 10);
+  // &.-sliding {
+  //   .chi-sidenav__list > li {
+  //     &.-active > a {
+  //       box-shadow: none;
+  //     }
+  //   }
+  // }
+
+  .chi-drawer {
+    &__header {
+      align-items: center;
+      color: set-color(grey, 70);
+      display: flex;
+      height: 4rem;
+      padding-left: 1.5rem;
+
+      .chi-btn {
+        &.-close {
+          right: 1rem;
+          top: 1rem;
+        }
+      }
     }
-  }
 
-  &.-sliding {
-    .chi-sidenav__list > li {
-      &.-active > a {
-        box-shadow: none;
+    &__content {
+      background: set-color(grey, 10);
+    }
+
+    &.-active {
+      .chi-drawer__content {
+        .chi-sidenav__drawer-list {
+          li {
+            a {
+              .chi-sidenav__title {
+                .chi-icon {
+                  margin-left: 0.5rem;
+                }
+              }
+            }
+
+            &.-active {
+              & > a {
+                background-color: set-color(grey, 90);
```

Comment:
This is the dark background-color causing the conflict in the base version. Change to `background: none;` and change the color to `color: $primary-color;`